protocol-designer/src/organisms/EditProtocolMetadataModal/index.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import { createPortal } from 'react-dom'
+import styled from 'styled-components'
+import { useForm } from 'react-hook-form'
+import {
+  BORDERS,
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  JUSTIFY_END,
+  Modal,
+  PrimaryButton,
+  SecondaryButton,
+  SPACING,
+  StyledText,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { getTopPortalEl } from '../../components/portals/TopPortal'
+import { InputField } from '../../components/modals/CreateFileWizard/InputField'
+import { actions, selectors as fileSelectors } from '../../file-data'
+import type { FileMetadataFields } from '../../file-data'
+
+interface EditProtocolMetadataModalProps {
+  onClose: () => void
+}
+export function EditProtocolMetadataModal(
+  props: EditProtocolMetadataModalProps
+): JSX.Element {
+  const { onClose } = props
+  const dispatch = useDispatch()
+  const { t } = useTranslation(['create_new_protocol', 'shared'])
+  const formValues = useSelector(fileSelectors.getFileMetadata)
+  const {
+    handleSubmit,
+    watch,
+    register,
+    formState: { isDirty },
+  } = useForm<FileMetadataFields>({ defaultValues: formValues })
+  const [protocolName, author, description] = watch([
+    'protocolName',
+    'author',
+    'description',
+  ])
+
+  const saveFileMetadata = (nextFormValues: FileMetadataFields): void => {
+    dispatch(actions.saveFileMetadata(nextFormValues))
+    onClose()
+  }
+
+  return createPortal(
+    <Modal
+      title={t('shared:edit_protocol_metadata')}
+      type="info"
+      closeOnOutsideClick
+      onClose={onClose}
+      footer={
+        <Flex
+          justifyContent={JUSTIFY_END}
+          gridGap={SPACING.spacing8}
+          padding={SPACING.spacing24}
```

Comment:
```suggestion
          padding={`0 ${SPACING.spacing24} ${SPACING.spacing24}`}
```

don't need padding-top because of ↓
![Screenshot 2024-09-05 at 5 41 33 PM](https://github.com/user-attachments/assets/5d362219-a712-4368-b425-290f9e0cb5fd)